src/SchemaApp.tsx

```diff
@@ -52,18 +52,31 @@ const RecentlyViewedMenu: React.FC<RecentlyViewedMenuProps> = (props) => {
   );
 };
 
+const examples = {
```

Comment:
Add an explicit type to this variable and this change looks great. Thanks for the submission!

Reply:
Done ✅